--- a/meson.build
+++ b/meson.build
@@ -275,21 +275,3 @@ configure_file(
   variables : pkgconfig_variables,
   libraries : [girepo_lib, glib_dep, gobject_dep],
 )
-
-# Dependency object used by Meson's GNOME module. This dependency variable must
-# be named girepo_dep for backward compatibility with projects that where already
-# using that name as fallback: dependency('gobject-introspection-1.0',
-# fallback : ['gobject-introspection', 'girepo_dep'])
-# FIXME: meson.override_dependency() and declare_dependency()'s variable arguments
-# are new in Meson 0.54.0, older versions of Meson won't be able to use g-i as
-# subproject anyway
-if meson.version().version_compare('>=0.54.0')
-  girepo_dep = declare_dependency(
-    sources: typelibs,
-    dependencies: girepo_dep,
-    variables: {
-      'girdir': meson.current_build_dir() / 'gir',
-    },
-  )
-  meson.override_dependency('gobject-introspection-1.0', girepo_dep)
-endif
