--- a/macros.in	2020-10-31 09:35:01.223407289 -0700
+++ b/macros.in	2020-10-31 09:36:20.206743556 -0700
@@ -560,10 +560,10 @@
 %_unique_debug_srcs	1

 # Whether rpm should put debug source files into its own subpackage
-#%_debugsource_packages	1
+%_debugsource_packages	1

 # Whether rpm should create extra debuginfo packages for each subpackage
-#%_debuginfo_subpackages 1
+%_debuginfo_subpackages 1

 #
 # Use internal dependency generator rather than external helpers?
